variables.tf

```diff
@@ -41,14 +43,72 @@ variable "ssh_private_key_path" {
   default     = ""
 }
 
+variable "tags" {
+  type        = list(any)
+  description = "String list of common tags for Equinix resources"
+  default     = ["k8s-cluster-cluster1", "k8s-nodepool-pool1"]
+}
+
+
+# Control Plane hosts vars
 variable "cp_ha" {
   description = "Whether to enable HA in Kubernetes control plane nodes"
   type        = bool
   default     = true
 }
 
-variable "worker_count" {
+variable "k8s_cluster_cp_hostname" {
+  description = "Hostname of each node in the control plane cluster"
+  type        = string
+  default     = "k8s-cluster1-pool1-cp"
+}
+
+variable "k8s_cluster_cp_plan" {
+  description = "Plan of the nodes in the control plane cluster"
+  type        = string
+  default     = "m3.small.x86"
+}
+
+variable "k8s_cluster_cp_os" {
+  description = "OS of the nodes in the control plane cluster"
+  type        = string
+  default     = "ubuntu_20_04"
+}
+
+variable "k8s_cluster_cp_billing_cycle" {
+  description = "Billing Cycle of the nodes in the control plane cluster"
+  type        = string
+  default     = "hourly"
+}
+
+
+# Data Plane hosts vars
+variable "dp_host_count" {
   description = "Number of worker nodes"
   type        = number
   default     = 1
 }
+
+variable "k8s_cluster_dp_hostname" {
```

Comment:
`dp` is better understood as "worker". The dp (data plane) naming hasn't really caught on outside of the one project that used this briefly before renaming it to working it.

Reply:
Thanks! Resolved in the commit [389a52b](https://github.com/equinix-labs/terraform-equinix-kubernetes-cluster/pull/29/commits/389a52bdd911f358899ee42c8bb8d7ee6d2aacef)